nimble/host/mesh/src/testing.c

```diff
@@ -123,7 +123,7 @@ void bt_test_print_credentials(void)
 	console_printf("IV Index: %08lx\n", (long) bt_mesh.iv_index);
 	console_printf("Dev key: %s\n", bt_hex(bt_mesh.dev_key, 16));
 
-	for (i = 0; i < ARRAY_SIZE(bt_mesh_cdb.app_keys); ++i)
+	for (i = 0; i < ARRAY_SIZE(bt_mesh_cdb.subnets); ++i)
```

Comment:
this looks kinda odd,   subnets here, but in loop app_key arrays is accessed

are lines 128-131 correct?

Reply:
I think this is now resolved.